CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Added Vowel Wabbit models
```

Comment:
Let's remove the `Added` (since it's redundant, Added, Added ... models) Sorry I still haven't added this to the docs #81